src/geocat/viz/util.py

```diff
@@ -639,7 +639,7 @@ def set_axes_limits_and_ticks(
 def truncate_colormap(cmap: matplotlib.colors.Colormap,
                       minval: typing.Union[int, float] = 0.0,
                       maxval: typing.Union[int, float] = 1.0,
-                      n: int = 100,
+                      num_values: int = 100,
```

Comment:
A few thoughts here:
* I'm a bit worried about changing this without letting folks know.  Despite not working as described (https://github.com/NCAR/geocat-viz/issues/124), this function and parameter are used in several examples over in the [gallery](https://github.com/search?q=repo%3ANCAR%2Fgeocat-examples%20truncate_colormap&type=code).  
* If we do end up changing this, n_segments, n_entries, or similar might be more appropriate.  It should roughly correspond to N in this matplotlib class: https://matplotlib.org/stable/api/_as_gen/matplotlib.colors.ListedColormap.html
* I think we might just want to deprecate this function after addressing the NCL/custom colormaps needs elsewhere.  Not sure if that impacts decision making.

Reply:
Great points Kaitlyn. Currently it is set up to accept `n` but raise a warning that it is changing, or `num_values` but you're right that might not be the best name. I think perhaps since we are reworking this function anyway -- I undo the changes to it and remove the rule that doesn't let us have one variable kwargs for now